src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/index_add_scatter_elements_update.cpp

```diff
@@ -209,12 +209,14 @@ class IndexAddTest : public testing::WithParamInterface<IndexAddTestParams>,
                 // All index values are expected to be within bounds [-d, d - 1] along dimension d pointed by axis.
                 auto d = dataShape[normalized_axis];
                 in_data.start_from = -1.0 * static_cast<int64_t>(d);
-                in_data.range = d-1;
+                // in_data.range = static_cast<uint32_t>(d-1 - in_data.start_from);
+                in_data.range = d - 1;
                 in_data.resolution = 1;
                 tensor = shape_size(indicesShape) == 0 ? ov::Tensor(funcInput.get_element_type(), indicesShape) :
                                             ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), indicesShape, in_data);
             } else if (i == 2) {  // "updates"
                 in_data.start_from = -50;
+                // in_data.range = 100;
```

Comment:
Looks like extra changes

Reply:
moved to another pr https://github.com/openvinotoolkit/openvino/pull/24682